eng/pipelines/build.yml

```diff
@@ -42,10 +42,10 @@ extends:
                 inputs:
                   useGlobalJson: true
                   performMultiLevelLookup: true
-              - task: NodeTool@0
-                displayName: "Install Node 20.x"
+              - task: UseNode@1
+                displayName: 'Install Node.js'
                 inputs:
-                  versionSpec: '20.x'
+                  version: $(NodeVersion)
```

Comment:
Personally I like what we did in the specs repo https://github.com/Azure/azure-rest-api-specs/blob/e278de5e90b0fee9cddd99ac3a5b30e1f3f6f137/eng/pipelines/typespec-apiview.yml#L27